chain/vm/runtime.go

```diff
@@ -351,16 +339,20 @@ func (rt *Runtime) Send(to address.Address, method abi.MethodNum, m vmr.CBORMars
 		params = buf.Bytes()
 	}
 
-	ret, err := rt.internalSend(rt.Message().Receiver(), to, method, value, params)
+	ret, err := rt.internalSend(rt.Receiver(), to, method, value, params)
 	if err != nil {
 		if err.IsFatal() {
 			panic(err)
 		}
 		log.Warnf("vmctx send failed: to: %s, method: %d: ret: %d, err: %s", to, method, ret, err)
-		return &dumbWrapperType{nil}, err.RetCode()
+		return err.RetCode()
 	}
 	_ = rt.chargeGasSafe(gasOnActorExec)
-	return &dumbWrapperType{ret}, 0
+
+	if err := out.UnmarshalCBOR(bytes.NewReader(ret)); err != nil {
+		rt.Abortf(exitcode.ErrSerialization, "failed to unmarshal return value: %s", err)
+	}
```

Comment:
This looks like it could be consensus-critical? I assume in the old code, the exit code would've been decided by however the caller handled an error arising from `ret.Into()`? 

In particular, I'm concerned that the return type might have been ignored in some `Send` calls, which would cloak the serialization error, whereas now it's guaranteed to be noticed in the internal send's exit code?

Reply:
We should only hit this if we have a bug. Honestly, we might want to panic (or do something _really_ noisy here).

Can we run a chain import against this to see if we have ever hit a bug here?